magento/pwa-studio
# [PWA-1588] Remove Enzyme

## Description

Remove Enzyme and React 17 Enzyme adapter.
Update tests using Enzyme.

## Related Issue

Closes https://jira.corp.magento.com/browse/PWA-1588.

## Acceptance

### Verification Stakeholders

## Verification Steps

#### Test scenario(s) for direct fix/feature

- Verify the @wojtekmaj/enzyme-adapter-react-17 package has been removed.
- Verify Enzyme as been removed.
- Verify there is appropriate test coverage for the removed Enzyme tests.
- Make sure no other tests were impacted.

## Breaking Changes

If users are using Enzyme in their projects, they will need to either update the affected tests.
